libarchive/archive_write_disk_windows.c

```diff
@@ -476,8 +476,16 @@ permissive_name_w(struct archive_write_disk *a)
 	}
 	else
 	{
```

Comment:
Please remove the extra brackets (and the extra indentation).

This block should look like
```
if (l > 3 && ...)
{
    ...
}
else if (l > 2 && ... )
{
   ...
}
else
{
   ...
}
```

Reply:
@kientzle Thanks for your feedback, I have updated the code.